src/Transport/HttpTransport.php

```diff
@@ -102,9 +102,9 @@ public function send(Event $event): ?string
      * Cleanups the pending requests by forcing them to be sent. Any error that
      * occurs will be ignored.
      */
-    private function cleanupPendingRequests(): void
+    public function await(): void
     {
-        foreach ($this->pendingRequests as $pendingRequest) {
+        while ($pendingRequest = array_pop($this->pendingRequests)) {
```

Comment:
Why this change? It seems just a memory/CPU overhead to me.

Reply:
Because I thought the pendingRequests needed to be popped (missed it cleans itself), it doesn't. So this is going to be reverted 👍